packages/circle-react-elements/src/components/FormErrorText/FormErrorText.stories.ts

```diff
@@ -0,0 +1,19 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import { FormErrorText } from './FormErrorText';
+
+const meta = {
+  title: 'FormErrorText',
+  component: FormErrorText,
+  tags: ['autodocs'],
+} satisfies Meta<typeof FormErrorText>;
+
+export default meta;
```

Comment:
FormErrorText is an internal component intended for use within our forms and not something that should be exported for external use or included in the storybook. It serves as a building block for the forms we export, but it doesn’t make sense to expose it as a standalone component. Does it make sense?

Reply:
fixed